src/utils/fetcher.ts

```diff
@@ -0,0 +1,5 @@
+export const fetcher = (url: string | null) => {
+  if (url) {
+    return fetch(url).then((res) => res.json())
```

Comment:
Implement proper `catch`, even if it is just console.error'ing the response.

In the future this will help us in implementing error messages.